frontend/controller/actions/identity.js

```diff
@@ -21,6 +21,22 @@ import type { Key } from '../../../shared/domains/chelonia/crypto.js'
 import { handleFetchResult } from '../utils/misc.js'
 import { encryptedAction } from './utils.js'
 
+const requireToSuccess = async (fnToSuccess) => {
+  if (typeof fnToSuccess !== 'function') {
+    return
+  }
+
+  let failureCount = 0
+  do {
+    try {
+      return await fnToSuccess()
+    } catch (err) {
+      console.log('[requireToSuccess:]', err)
```

Comment:
errors should always be logged with `console.error`, however I'm not sure this function should exist at all (see comment below)